src/value_converters/collection_appender.cc

```diff
@@ -39,8 +39,8 @@ void collection_appender_helper<double>::append_to_collection(CassCollection *co
 }
 
 template <>
-void collection_appender_helper<const std::string &>::append_to_collection(CassCollection *collection,
-                                                                           const std::string &value) {
+void collection_appender_helper<std::string>::append_to_collection(CassCollection *collection,
+                                                                           std::string value) {
```

Comment:
Why copy instead of const references? Were there some compatibility issues with other specifications?